stories/5-components/Brand/AuIcon.stories.js

```diff
@@ -42,6 +42,17 @@ const Template = (args) => ({
   context: args,
 });
 
+const CustomIconTemplate = (args) => ({
+  template: hbs`
+    <AuIcon
+      @icon={{component 'au-icon' icon=this.icon}}
```

Comment:
This example feels a bit silly since there is no real reason to do this 😄. I wouldn't add the example yet. I'll probably add something as part of #483 or when we consider this feature no longer experimental.

Reply:
True, but I thought it was better than no example. The alternative was to add a component just for this example, but I thought that might be even more confusing...

Reply:
Yea, I would have gone for a custom example component in the dummy app. Maybe we are just confused by different things 😄. Anyways, I would split it off or remove it for now.